File: 02_ProgrammingFundamentals/_03_Arrays/_03_MoreExercises/_02_PascalTriangle.java
package _02_ProgrammingFundamentals._03_Arrays._03_MoreExercises;

import java.util.Scanner;

public class _02_PascalTriangle {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input value
        int n = Integer.parseInt(scan.nextLine());

        // Print pascal triangle
        for (int i = 0; i < n; i++) {
            int number = 1;
            
            for (int j = 0; j <= i; j++) {
                System.out.print(number + " ");
                number = number * (i - j) / (j + 1);
            }

            System.out.println();
        }

        scan.close();
    }
}

File: 24_C#Fundamentals/_03_Arrays/_03_MoreExercises/_02_PascalTriangle/_02_PascalTriangle.cs
using System;

public class _02_PascalTriangle
{
    public static void Main(string[] args)
    {
        // Read input value
        int n = int.Parse(Console.ReadLine());

        // Print pascal triangle
        for (int i = 0; i < n; i++)
        {
            int number = 1;

            for (int j = 0; j <= i; j++)
            {
                Console.Write(number + " ");
                number = number * (i - j) / (j + 1);
            }

            Console.WriteLine();
        }
    }
}
